src/app/project/list/list-item-form/list-item-form.component.ts

```diff
@@ -90,6 +93,14 @@ export class ListItemFormComponent implements OnInit {
 
     @Output() refreshParent: EventEmitter<ListNodeOperation> = new EventEmitter<ListNodeOperation>();
 
+    // permissions of logged-in user
+    session: Session;
+    sysAdmin = false;
+    projectAdmin = false;
+
+    // project data
+    project: ReadProject;
```

Comment:
I don't think this variable is needed, you can use a `const` to temporarily store the project you get from the cache

Reply:
You are right, it is not needed elsewhere